src/components/Sweep/useSweep.ts

```diff
@@ -0,0 +1,37 @@
+import Web3 from 'web3';
```

Comment:
this file is not named or implemented correctly. by convention, any files / functions named `use...` refer to hooks, but this code has nothing to do with a hook. if all you need are contract functions and you don't want / need a hook, you can rename this file to e.g. `magicContractHelper`